src/js/components/Select/Select.js

```diff
@@ -128,6 +130,8 @@ const Select = forwardRef(
     }, [value, valueKey]);
     // search input value
     const [search, setSearch] = useState();
+    // focus state
+    const [focus, setFocus] = useState();
```

Comment:
Why do we need to add this? I don't think DropButton or Button take a `focus` property.

Reply:
Ahh I think this was just part of when I was "brute-forcing" my way by following how other components implemented their own focus states. After reviewing the code this is most definitely not necessary and will be removed in the upcoming update.